stonesoup/predictor/kernel.py

```diff
@@ -0,0 +1,82 @@
+import copy
+
+import numpy as np
+
+from ._utils import predict_lru_cache
+from .kalman import KalmanPredictor
+from ..base import Property
+from ..kernel import Kernel, QuadraticKernel
+from ..types.prediction import Prediction
+from ..types.state import State
+from ..types.update import KernelParticleStateUpdate
+
+
+class AdaptiveKernelKalmanPredictor(KalmanPredictor):
+    """AdaptiveKernelKalmanFilter class
```

Comment:
would be useful to expand on this docstring to be consistent with documentation for other predictors.